app/functions/100ms/notificationReminder/index.js

```diff
@@ -0,0 +1,172 @@
+    const videoConsultation = await knex("video_consultations")
+      .whereIn("request_status", ["buyer_payment_done"])
+      .where("type", "future_date");
+
+    for (i = 0; i < videoConsultation.length; i++) {
+      const consultationRequest = videoConsultation[i];
+
+      var videoConsulationdate = JSON.stringify(
+        consultationRequest.query_date_time
+      ).replaceAll('"', "");
+
+      var convertedConsultationDate = moment(videoConsulationdate).format(
+        "YYYY-MM-DD HH:mm:ss"
+      );
+
+      var timeForConsultationInHours = moment(convertedConsultationDate).diff(
+        moment(tzTime),
+        "hours"
+      );
+
+      // console.log("hours difference", timeForConsultationInHours);
+      var minutesDiff = moment(convertedConsultationDate).diff(
+        moment(tzTime),
+        "minutes"
+      );
+      // console.log("Minutes:", minutesDiff);
+
+      let bankerProfile = await Profile.findOne({
+        where: {
+          uuid: consultationRequest.banker_uuid,
+          type: "fm-seller",
+        },
+      });
+
+      let buyerProfile = await Profile.findOne({
+        where: {
+          uuid: consultationRequest.buyer_uuid,
+          type: "fm-buyer",
+        },
+      });
+
+      // sending notification to banker and buyer before 24 hr of the consultation
+      if (timeForConsultationInHours === +24 && minutesDiff === +1440) {
+        console.log(" we are inside sending inside if condition");
+
+        // send reminder to the banker
+        await sendPushNotification({
+          event_type: "scheduled_consultation_reminder",
+          user_id: bankerProfile.user_uuid,
+          data: {
+            time: timeForConsultationInHours,
+            nameOne: bankerProfile.data.full_name,
+            nameTwo: buyerProfile.data.full_name,
+            profile_uuid: bankerProfile.uuid,
+            notification_type: "video_consultation_reminde",
+          },
+          ignore_user_contacts: false,
+          contact_infos: [
+            {
+              type: "email",
+              value: bankerProfile.data.email,
+            },
+          ],
+        });
+
+        //send reminder to buyer
+
+        await sendPushNotification({
+          event_type: "scheduled_consultation_reminder",
+          user_id: buyerProfile.user_uuid,
+          data: {
+            time: timeForConsultationInHours,
+            nameOne: buyerProfile.data.full_name,
+            nameTwo: bankerProfile.data.full_name,
+            profile_uuid: buyerProfile.uuid,
+            notification_type: "video_consultation_reminder",
+          },
+          ignore_user_contacts: false,
+          contact_infos: [
+            {
+              type: "email",
+              value: buyerProfile.data.email,
+            },
+          ],
+        });
+      }
+
+      // sending notification to banker and buyer before 1 hr of the consultation
+      else if (timeForConsultationInHours === +1 && minutesDiff === +60) {
```

Comment:
@ChaitanyaV98 what is +1 here?

Reply:
actually `timeForConsultationInHours` and `minutesDiff` are of number type, so thats why +1, +60 etc.. which will be considered as number data type.